src/public/modules/forms/viewmodels/Fields/CheckboxField.class.js

```diff
@@ -17,6 +17,12 @@ class CheckboxField extends MixIns.RangeValidation(ArrayAnswerField) {
 
   getResponse() {
     const response = super.getResponse()
+    // Throw error if field value is missing
+    if (this.isVisible && this.required && this.fieldValue.every((v) => !v)) {
```

Comment:
Should this be in ArrayAnswerField class?

Reply:
actually I don't think so, because checkboxes have `string[]` values whereas tables have `string[][]` values so the checks would be different. I added this check to `CheckboxField` because tables consist of smaller `SingleAnswerField`s, so the changes to `SingleAnswerField` also cover tables.